openid-federation-1_0.xml

```diff
@@ -3247,12 +3247,14 @@
           Trust Marks.
         </t>
         <t>
-          Trust Mark JWTs MUST be explicitly typed by setting the
-          <spanx style="verb">typ</spanx> header parameter to
-          <spanx style="verb">trust-mark+jwt</spanx> to prevent
-          cross-JWT confusion, per Section 3.11 of <xref target="RFC8725"/>.
-	  Trust Marks without a <spanx style="verb">typ</spanx> header parameter
-	  or with a different <spanx style="verb">typ</spanx> value MUST be rejected.
+          Trust Mark JWTs MUST include the <spanx style="verb">typ</spanx> header 
+          parameter. The supported <spanx style="verb">typ</spanx> values SHOULD be 
+          defined by trust frameworks to prevent cross-JWT confusion, as outlined in 
+          Section 3.11 of <xref target="RFC8725"/>. Trust Marks without the 
+          <spanx style="verb">typ</spanx> header or with <spanx style="verb">typ</spanx> 
+          set to <spanx style="verb">JWT</spanx> MUST be rejected. All implementations 
+          MUST support the <spanx style="verb">trust-mark+jwt</spanx> type value, 
+          ensuring a common and recognized typ value for issuing Trust Marks.
```

Comment:
```suggestion
          Trust Mark JWTs MUST be explicitly typed by using the
          <spanx style="verb">typ</spanx> header parameter to prevent
          cross-JWT confusion, per Section 3.11 of <xref target="RFC8725"/>.
          The <spanx style="verb">typ</spanx> header parameter value MUST be
          <spanx style="verb">trust-mark+jwt</spanx> 
          unless the trust framework in use defines a more specific
          media type value for the particular kind of Trust Mark.
	  Trust Marks without a <spanx style="verb">typ</spanx> header parameter
	  or an unrecognized <spanx style="verb">typ</spanx> value MUST be rejected.
```

Refactor the description to be more like other `typ` descriptions, while still allowing trust frameworks to define more specific media type values.